src/routes/index.ts

```diff
@@ -70,10 +76,22 @@ export class Index {
                     res.setHeader('Content-Type', 'application/'+req.query.format)
                     res.status(200).send(this.dbConn.format_data(data, req.query.format));
                   }).catch((error) => {
-                    res.status(500).send(error);
+                    if(req.query.format=="xml"){
+                      res.setHeader('Content-Type', 'application/xml')
+                    }
+                    if(req.query.format=="json" || req.query.format==null){
+                      res.setHeader('Content-Type', 'application/json')
+                    }
+                    res.status(500).send(this.dbConn.format_error(error, req.query.format));
```

Comment:
@akmiller01 this is all fine. A code style note those ... could you extract this duplicated section of code into a function that takes in the request, response & error. Should cut down on the lines.